packages/shared/src/components/squads/SquadHeaderBar.tsx

```diff
@@ -0,0 +1,63 @@
+import classNames from 'classnames';
+import React, { HTMLAttributes, ReactElement } from 'react';
+import { Button } from '../buttons/Button';
+import MenuIcon from '../icons/Menu';
+import AddUserIcon from '../icons/AddUser';
+import { ProfilePicture } from '../ProfilePicture';
+import { SimpleTooltip } from '../tooltips/SimpleTooltip';
+import useSquadMenu from '../../hooks/useSquadMenu';
+import SquadHeaderMenu from './SquadHeaderMenu';
+import { SquadMember } from '../../graphql/squads';
+
+type SquadHeaderBarProps = {
+  members: SquadMember[];
+  memberCount: number;
+} & HTMLAttributes<HTMLDivElement>;
+
+export function SquadHeaderBar({
+  members,
+  memberCount,
+  className,
+  ...props
+}: SquadHeaderBarProps): ReactElement {
+  const { onMenuClick } = useSquadMenu();
+  return (
+    <div className={classNames('flex flex-wrap gap-4', className)} {...props}>
+      <div className="flex items-center px-0.5 rounded-14 border border-theme-divider-secondary">
```

Comment:
Tsahi changed the guideline on this, the sketch is updated with the new flow.
It now has a border in between